provider/ec2/internal/ec2instancetypes/instancetypes_test.go

```diff
@@ -26,7 +26,6 @@ func (s *InstanceTypesSuite) TestRegionInstanceTypes(c *gc.C) {
 		"m3.medium", "m3.large", "m3.xlarge", "m3.2xlarge",
 		"c1.medium", "c1.xlarge", "cc2.8xlarge",
 		"c3.large", "c3.xlarge", "c3.2xlarge", "c3.4xlarge", "c3.8xlarge",
-		"cg1.4xlarge",
 		"g2.2xlarge",
```

Comment:
Unrelated to this change directly I know, just some questions. TestSupportsClassic() still uses "cg1.4xlarge": assertSupportsClassic("cg1.4xlarge").  Is it correct that the test still passes?  I'm wondering if the test is testing what it thinks it's testing?

Reply:
I agree, I think some of the tests are setup via mocks and therefore some tests still pass even when they're not "live". I don't fully grasp why I've had to change so many tests, so it would be good to get that explained.